apps/mdn/mdn-aws/k8s/kumascript.deploy.yaml.j2

```
@@ -60,11 +45,22 @@ spec:
               value: "{{ NEW_RELIC_LOG }}"
             - name: NEW_RELIC_BROWSER_MONITOR_ENABLE
               value: "{{ NEW_RELIC_BROWSER_MONITOR_ENABLE }}"
-            - name: newrelic__license_key
+            - name: NEW_RELIC_HIGH_SECURITY
+              value: "{{ NEW_RELIC_HIGH_SECURITY }}"
```

If you decide [to&nbsp;hardcode this&nbsp;to&nbsp;`true`](#discussion_r249180578):
```suggestion
              value: "true"
```